start-finalSetupWorld

```diff
@@ -51,7 +51,7 @@ if [[ "$WORLD" ]] && ( isTrue "${FORCE_WORLD_COPY}" || [ ! -d "$worldDest" ] );
       [ -d "${baseDir}_nether" ] && rsync --remove-source-files --recursive --delete "${baseDir}_nether/" "${worldDest}_nether"
       [ -d "${baseDir}_the_end" ] && rsync --remove-source-files --recursive --delete "${baseDir}_the_end/" "${worldDest}_the_end"
     fi
-  else
+  elif [[ "$WORLD" ]]; then
```

Comment:
By itself this change makes sense; however, the outer conditionally of this blocks already tests for `if [[ "$WORLD" ]]`

https://github.com/itzg/docker-minecraft-server/blob/861faaa2857f5052ff46375dc0e30ea109c43460/start-finalSetupWorld#L16

so there must have been something else going on to see the behavior you did.

Reply:
...and `WORLD` is not set by default.